v0/destinations/tiktok_ads/transform.js

```diff
@@ -110,9 +100,48 @@ const trackResponseBuilder = async (message, { Config }) => {
   response.method = defaultPostRequestConfig.requestMethod;
   response.endpoint = TRACK_ENDPOINT;
   response.body.JSON = removeUndefinedAndNullValues(payload);
+
   return response;
 };
 
+const trackResponseBuilder = async (message, { Config }) => {
+  const pixel_code = Config.pixelCode;
+  const eventsToStandard = Config.eventsToStandard;
+
+  let event = get(message, "event");
+  event = event ? event.trim().toLowerCase() : event;
+  if (!event) {
+    throw new CustomError("Event name is required", 400);
+  }
+
+  const standardEventsMap = getHashFromArrayWithDuplicate(eventsToStandard);
+  const trimmedEvent = event.toLowerCase().trim();
+
+  if (
+    eventNameMapping[event] === undefined &&
+    !standardEventsMap[trimmedEvent]
+  ) {
+    throw new CustomError(`Event name (${event}) is not valid`, 400);
+  }
+
+  const returnArray = [];
+  if (standardEventsMap[trimmedEvent]) {
+    Object.keys(standardEventsMap).forEach(key => {
+      if (key === trimmedEvent) {
+        standardEventsMap[trimmedEvent].forEach(val => {
+          returnArray.push(getTrackResponse(message, Config, val));
+        });
+      }
+    });
+    return returnArray;
+  }
+
+  const response = defaultRequestConfig();
```

Comment:
response is unused variable is this correct?

Reply:
Yes, removed it